uplc-semantics.md

```diff
@@ -10,8 +10,11 @@ module UPLC-SEMANTICS
   imports K-EQUAL
   imports KRYPTO
   imports BYTESTRING
+  imports STRING-BUFFER
+  imports BYTES
 
   syntax AClosure ::= Clos(Value, Map)
+  syntax String ::= #sha3(String) [function]
```

Comment:
I don't understand why this line is being added here. Could you clarify? I'm looking for `#sha3` and I can't find it anywhere.

Reply:
I will remove it.